docusaurus/docs/learn/introduction/_components.md

```diff
@@ -15,7 +15,7 @@ The OpenZiti Controller must be configured with public key infrastructure
 (PKI). The configured PKI is used to create secure, mutually-
 authenticated TLS (mTLS) network connections between any two
 pieces of the OpenZiti Network. The OpenZiti Controller does not provide its
-own PKI but for the OpenZiti Controller to sign certificate requests (CSR)
+own PKI but for the OpenZiti Controller to sign Certificate Signing Request (CSR)

```

Comment:
thanks. I think on this one it probably should stay lower-case and plurarize 'request'. I checked wikipedia and they don't propercase "certificate signing request".

Reply:
or - "to sign a certificate signing request". That makes sense too

Reply:
:+1: